plugins/sdm/lib/helper/base_grant_helper.py

```diff
@@ -91,7 +91,8 @@ def __notify_access_request_entered(self, sender_nick, resource_name, request_id
         operation_desc = self.get_operation_desc()
         formatted_resource_name, formatted_sender_nick = self.__bot.format_access_request_params(resource_name, sender_nick)
         yield f"Thanks {formatted_sender_nick}, that is a valid request. Let me check with the team admins: {team_admins}\nYour request id is **{request_id}**"
-        self.__notify_admins(f"Hey I have an {operation_desc} request from USER {formatted_sender_nick} for {self.__grant_type.name} {formatted_resource_name}! To approve, enter: **yes {request_id}**", message)
+        self.__notify_admins(f"Hey I have an {operation_desc} request from USER {formatted_sender_nick} for {self.__grant_type.name} {formatted_resource_name}!" +
+                             f" To approve, enter: **yes {request_id}**. To deny with a reason, enter: **no {request_id} your-reason**", message)
```

Comment:
Change: `To deny with a reason, enter: **no {request_id} your-reason**`
By: `To deny, enter: **no {request_id} [optional-reason]**`